automatic/brave/tools/helpers.ps1

```diff
@@ -0,0 +1,15 @@
+[array]$key = Get-UninstallRegistryKey -SoftwareName 'Brave*'
```

Comment:
Can you please change the code in this file to be inside a function instead of running immediately when the scripts run?

It would make it easier in the future if there is a need to extend this file with more helper functions.